development/extract_strings_from_objects.py

```diff
@@ -161,12 +161,13 @@ def add_line(value, component, sep, key, filename):
 			text = u'_("{value}")'.format(value=value[2:])
 			component = component + sep + str(parse_token(key, 'TIER'))
 			filename = filename.rsplit('.yaml')[0].split(OBJECT_PATH)[1].replace('/',':')
-			comment = '%s of %s' %(component, filename)
-			object_strings.append('# %s' %comment + ROWINDENT + '%-30s: %s' % (('"%s"') % component, text))
+			comment = u'{0!s} of {1!s}'.format(component, filename)
+			object_strings.append('# {comment!s}{rowindent}{component:<30}: {text!s}'.
+				format(comment=comment, rowindent=ROWINDENT, component='"{}"'.format(component), text=text))
```

Comment:
That we have discus: 

https://github.com/unknown-horizons/unknown-horizons/pull/2412#issuecomment-240862806
%-30s means every time 30 long right will fill with space.
I I remove -30s and the result is better for reading in the file
horizons/i18n/objecttranslations.py.

You should not change the output when changing to format(). If the output should be changed, open a new pull request for or at least do it in a different commit.